dataload/import_to_elasticsearch.py

```diff
@@ -228,8 +228,12 @@ def maybe_create_index(index_name=ES_INDEX):
 
     # Create it
     result = es.indices.create(index=index_name, body={"mappings": mappings, "settings": settings}, ignore=[400])
-    if 'error' in result and result['error']['reason'] == 'already exists':
-        print('Updating existing index')
+    if 'error' in result:
```

Comment:
@michaelwood does this also catch / should we also catch circumstances where we can't talk to ES, temporarily?

Reply:
The elasticsearch python library via requests/urllib will raise an exception if something goes wrong at the lower layers. Whereas this is just to raise an exception if the http request were sent and received and what was sent back was a json document with the error. 

https://github.com/elastic/elasticsearch-py/blob/master/elasticsearch/client/indices.py#L99